### ISSUE TITLE
Changing 'Playing' status to 'Streaming' Status.

### ISSUE THREAD

**Issue by OfficialTehlo** [DJArghlex/twitch-notifier-bot#18]
So as the title says, I am trying to set the 'Playing' to 'Streaming' as it does for some other bots.
How can I change this?

**Comment by pundm** [comment]
Have you checked [setPresence](https://izy521.github.io/discord.io-docs/Discord.Client.html#setPresence)? 

corresponding lines of code are nr. 433 - 437 currently

**Comment by OfficialTehlo** [comment]
Yes I have and its working now, but I've ran into another issue. Seems like OP ain't here no more sadly.

**Comment by pundm** [comment]
Is it because the v3 API was shutdown? If yes, see my issue post.

**Comment by OfficialTehlo** [comment]
Afaik that isn't my issue _yet_ but right now it seems that some commands don't work in order for me to set it up. Could you help out @pundm ?

**Comment by OfficialTehlo** [comment]
It's basically this:
https://github.com/DJArghlex/twitch-notifier-bot/issues/17

**Comment by pundm** [comment]
Ok np you have to edit twitchConfig.json manually {"streamers":{"Lirik":{"ChannelID":true}},**"servers":{"ServerID":"ChannelID"}**}

ServerID  = which server it is on to advertise the stream
ChannelID =channel it advertises in

you have to enable deveoloper options in discord and then you can get the IDs on right click,

**Comment by OfficialTehlo** [comment]
And do I need to change anything else like your API fix? Or should your comment above fix it?

**Comment by OfficialTehlo** [comment]
Is it possible to add you on Discord @pundm ? Makes it a lot easier since I kinda need some more help with the bot since it doesn't seem to be working.

**Comment by pundm** [comment]
rotunde#3461

**Comment by pundm** [comment]
rotunde#3461

**Comment by OfficialTehlo** [comment]
Added!
